src/backend/joanie/payment/__init__.py

```diff
@@ -16,3 +17,38 @@ def get_payment_backend():
             "JOANIE_PAYMENT_BACKEND configuration seems not valid. "
             "Check your settings.py."
         ) from error
+
+
+def get_country_calendar():
+    """
+    Instantiate the contract's calendar through `JOANIE_CONTRACT_COUNTRY_CALENDAR` setting.
+    """
+    calendar_path = getattr(settings, "JOANIE_CONTRACT_COUNTRY_CALENDAR", None)
+    if not calendar_path:
+        raise ValueError(
+            "`JOANIE_CONTRACT_COUNTRY_CALENDAR` setting must be defined in your settings.py."
+        )
+    try:
+        return import_string(calendar_path)()
+    except (AttributeError, ImportError) as error:
+        raise ImproperlyConfigured(
+            "Cannot instantiate a calendar. "
+            "`JOANIE_CONTRACT_COUNTRY_CALENDAR` configuration seems not valid. "
+            "Check your settings.py"
+        ) from error
+
+
+def get_withdrawal_period_days():
+    """
+    Retrieve the number for the period of days for the withdrawal period when signing a contract
+    through `JOANIE_CONTRACT_WITHDRAWAL_PERIOD_DAYS` setting.
+    """
+    withdrawal_period_days = getattr(
+        settings, "JOANIE_CONTRACT_WITHDRAWAL_PERIOD_DAYS", None
+    )
+    if not withdrawal_period_days or not isinstance(withdrawal_period_days, int):
+        raise ValueError(
+            "`JOANIE_CONTRACT_WITHDRAWAL_PERIOD_DAYS` must be defined "
+            "as an integer value in your settings."
+        )
+    return withdrawal_period_days
```

Comment:
This function is useless IMO. In the settings use are using a `PositiveIntegerValue` so there is no reason you have something else than an integer here. Using directly `settings.JOANIE_CONTRACT_WITHDRAWAL_PERIOD_DAYS` where you need it is enough

Reply:
Thanks for the feedback, I also share your opinion. Let's go for it.